x/evm/migrations/migrate_deliver_tx_gas_limit.go

```diff
@@ -0,0 +1,15 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/evm/keeper"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+func MigrateDeliverTxHookWasmGasLimitParam(ctx sdk.Context, k *keeper.Keeper) error {
+	defaultParams := types.DefaultParams()
+	defaultDeliverTxHookWasmGasLimit := defaultParams.DeliverTxHookWasmGasLimit
+	defaultParams.DeliverTxHookWasmGasLimit = defaultDeliverTxHookWasmGasLimit
+	k.SetParams(ctx, defaultParams)
```

Comment:
This will still override the other existing params, instead of starting from default params, we need to load the existing params, and set these two values in that struct